src/main/java/study/racinggame/domain/RaceGame.java

```diff
@@ -36,7 +34,7 @@ public boolean isEndGame() {
     public List<Car> winners() {
         List<Car> winners = new ArrayList<>();
         cars.getCars().forEach((car) -> {
-            if (car.isMaxPosition(maxPosition)) {
+            if (car.getPosition().isMaxPosition(maxPosition)) {
```

Comment:
```suggestion
            if (car.isMaxPosition(maxPosition)) {
```
positon을 get하지 말고 Car에 위와 같이 메시지를 보내면 어떨가?
Car의 isMaxPosition() 메서드는 이 작업을 position의 isMaxPosition()에 위임하는 구조.